### PR TITLE
Focus Android 7.0 strings

### PR DESCRIPTION
**Pull request by boek**

### CODE REVIEW

File: locales/templates/app.pot [hunk 1]
```diff
@@ -1202,3 +1252,160 @@ msgctxt "custom_tab_copy_url_action"
...
+
+#. Description for the prompt to enable search suggestions         %1$s will
+#. be replaced by the app name (e.g. Firefox Focus)         %2$s will be
+#. replaced with "Learn more" which will be a tappable link         using the
+#. string: "enable_search_suggestion_subtitle_learnmore"
+#, c-format
+msgctxt "enable_search_suggestion_subtitle"
+msgid ""
+"To get search suggestions, %1$s needs to send what you type in the "
+"address bar to the search engine you've chosen. %2$s."
```
Comment (Delphine) [inline locales/templates/app.pot]: Sorry didn't spot this the first time. As per localization guidelines, needs to be a proper apostrophe:  ’ (smart quotes). It's ok not to use them in the localization notes, but not in the actual strings.

File: locales/templates/app.pot [hunk 2]
```diff
@@ -1202,3 +1252,160 @@ msgctxt "custom_tab_copy_url_action"
...
+"Site behaving unexpectedly?\n"
+" Try turning off Tracking Protection"
+msgstr ""
+
+#. Tip displayed on home view explaining how to add sites to homescreen
+#. The new line here must be kept as the second half of the string is clickable
+#. for the user.
+msgctxt "tip_add_to_homescreen"
+msgid ""
+"Get one-tap access to sites you use most\\n\n"
```
Comment (Delphine) [inline locales/templates/app.pot]: Shouldn't this just be \n rather?

File: locales/templates/app.pot [hunk 3]
```diff
@@ -1202,3 +1252,160 @@ msgctxt "custom_tab_copy_url_action"
...
+"Open a link in a new tab\n"
+" Long-press any link on a page"
+msgstr ""
+
+#. Tip displayed on home view explaining how to request the desktop version of
+#. a website.      The new line here must be kept as the second half of the
+#. string is clickable for the user.
+msgctxt "tip_request_desktop"
+msgid ""
+"Rather see the full desktop site?\\n\n"
```
Comment (Delphine) [inline locales/templates/app.pot]: Same here, I think this should be just \n? (correct me if I'm wrong though :) )

**Review by Delphine — CHANGES_REQUESTED** [review]
I think we're almost there! Just one comment and a couple questions :) thanks!

**Comment by boek** [comment]
@Delphine fixed!

**Review by Delphine — APPROVED** [review]
Seems like we're good to go!
Thanks Jeff

**Pull request merged**